Dockerfile

```diff
@@ -27,7 +27,7 @@ FROM ${TOOLCHAIN} AS toolchain
 RUN apk --update --no-cache add bash curl build-base protoc protobuf-dev
 
 # build tools
-FROM toolchain AS tools
+FROM --platform=${BUILDPLATFORM} ${TOOLCHAIN} AS tools
```

Comment:
this should probably be still `toolchain` not, `${TOOLCHAIN}, ? and `--platform` might be set on the first `FROM ${TOOLCHAIN} AS toolchain`

Reply:
yes, copy paste error, thanks